kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/rest/jaxb/DynamicJaxbContext.java

```diff
@@ -86,27 +70,22 @@
 
  * <b>In all cases, please preserve existing tests when modifying this class!</b>
  */
-@ApplicationScoped
+
 public class DynamicJaxbContext extends JAXBContext {
 
     private static final Logger logger = LoggerFactory.getLogger(DynamicJaxbContext.class);
 
     // The contextsCache needs to be a ConcurrentHashMap because parallel operations (involing *different* deployments) can happen on it.
-    private static ConcurrentHashMap<String, JAXBContext> contextsCache = new ConcurrentHashMap<String, JAXBContext>();
+    static ConcurrentHashMap<String, JAXBContext> contextsCache = new ConcurrentHashMap<String, JAXBContext>();
     static {
         setupDefaultJaxbContext();
```

Comment:
similar for this, if that could be moved to manager class then it could be done as part of @PostConstruct method instead of static block